Task50/Task50/DownloadingAndRefreshingTest.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Support.UI;
+
+namespace Task50
+{
+    [TestFixture]
+    public class DownloadingAndRefreshingTest
+    {
+        private static readonly string URL = "https://www.seleniumeasy.com/test/bootstrap-download-progress-demo.html";
+        private IWebDriver _driver;
+
+
+        [SetUp]
+        public void Setup()
+        {
+            _driver = new ChromeDriver {Url = URL};
+            _driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(9);
+        }
+
+        [TearDown]
+        public void TearDownTest()
+        {
+            _driver.Quit();
+        }
+
+        [Test]
+        public void ProgressBarForDownloadTest()
+        {
+            _driver.FindElement(By.Id("cricle-btn")).Click();
+
+            var wait = new WebDriverWait(_driver, TimeSpan.FromSeconds(15));
+            var element = wait.Until(condition =>
```

Comment:
Use more understandable name to reflect content of wait method. For example, isProgressOfDownloadingPassed.